lms/templates/edraak_forus/message.html

```diff
@@ -0,0 +1,29 @@
+<%page expression_filter="h"/>
+<%! from django.conf import settings %>
+<%! from django.utils.translation import ugettext as _ %>
+<%! from openedx.core.djangolib.markup import HTML, Text %>
+
+<%inherit file="../main.html" />
+
+<section class="outside-app">
+  <h1>
+    ${_('Sorry, we are unable to enroll you in the course.')}
+  </h1>
+  <p class="text-center">
+      ${Text(message)}
+  </p>
+
+  <hr />
+
+  <p class="text-center">
+    ${Text(_('Please go to {forus_link}, and try again. If the problem persists, please email us at {email}.')).format(
+        forus_link=HTML(u'<a href="{url}">{name}</a>').format(
```

Comment:
If you're using `HTML`, use `Text` for it's params. `Cmd+Shift+F` for examples :rofl: